lib/matplotlib/axes/_axes.py

```diff
@@ -6177,9 +6177,15 @@ def pcolorfast(self, *args, alpha=None, norm=None, cmap=None, vmin=None,
         non-regular rectangular or arbitrary quadrilateral).
 
         .. warning::
+           C can in some cases be 3D with the last dimension as rgb(a).
+           This is available when C qualifies for image or pcolorimage type,
+           otherwise if C is 3D, a TypeError will be raised.
 
-           This method is experimental. Compared to `~.Axes.pcolor` or
-           `~.Axes.pcolormesh` it has some limitations:
+        ``ax.pcolorfast(C, **kwargs)`` is equivalent to
+        ``ax.pcolorfast([0,nc], [0,nr], C, **kwargs)``
+
+        This method is experimental. Compared to `~.Axes.pcolor` or
```

Comment:
keep these two lines indented (basically, undo the change)

Reply:
Fixed in https://github.com/matplotlib/matplotlib/pull/8690/commits/af603a93495230161eb3c2354236f986b15769f8